src/test/compile-fail/E0259.rs

```diff
@@ -10,5 +10,6 @@
 
 extern crate collections;
 extern crate libc as collections; //~ ERROR E0259
+//~^ already imported
```

Comment:
@EugeneGonzalez - please change this label to be include NOTE as shown in the [blog post](http://www.jonathanturner.org/2016/08/helping-out-with-rust-errors.html)